### ISSUE TITLE
Logistics pipes being scanned as a trojan?

### ISSUE THREAD

**Issue by DangerMage** [RS485/LogisticsPipes#1657]
So My friend's bit defender scanned and said it was a trojan and so i uploaded a fresh download of the same 1.7.10 version from curseforge and put it into virus total and the same thing popped up but with 10 AVs:

https://www.virustotal.com/gui/file/fe0750fbc59cba524437133013727bb58edba88cc6154fe9b97dfe1a5d8916f7

Most recent one for 1.12.2 is fine: https://www.virustotal.com/gui/file/7407e42796459e8ecb66fc7be3cecb64a8e691864b5e7c5afabaffd05994f9d3

**Comment by masterqgj** [comment]
Same here on my PC and one of my mates playing on my server. My mates are a bit worried about that :(.

**Comment by metzen227** [comment]
My windows defender also identified an old version of the mod 1.7.10  0.9.4.5.4 as a trojan.
Wacatac.B!ml to be exact.  Has there been any update on this?

**Comment by DangerMage** [comment]
> My windows defender also identified an old version of the mod 1.7.10 0.9.4.5.4 as a trojan. Wacatac.B!ml to be exact. Has there been any update on this?

No updates so far.

**Comment by lwh9346** [comment]
Have the same problem too.

**Comment by ghost** [comment]
TL;DR false positive

The same problem occurred and I decompiled the mod. The problem is EnderIOProxy, which contains 2 versions of bytecode. Because of the difficulty to decompile this bytecode, the antivirus considers it a virus. I can't say exactly what it does, but it doesn't do any harm by looking at the bytecode.

Screenshots of the bytecode from Sublime Text (lol)
https://ibb.co/xX1tQSJ
https://ibb.co/bQNHYYH

**Comment by DangerMage** [comment]
> TL;DR false positive
> 
> The same problem occurred and I decompiled the mod. The problem is EnderIOProxy, which contains 2 versions of bytecode. Because of the difficulty to decompile this bytecode, the antivirus considers it a virus. I can't say exactly what it does, but it doesn't do any harm by looking at the bytecode.
> 
> Screenshots of the bytecode from Sublime Text (lol) https://ibb.co/xX1tQSJ https://ibb.co/bQNHYYH

Oh good
